includes/database_updates.php

```diff
@@ -68,3 +68,26 @@ function db_update_remove_all_is_approximate_location_meta() {
     };
   };
 };
+
+
+// Function: Add approximate value to locations when we can do so without doing a new geocode
+// May want do a geocode sometime in a later version of the plugin
+if (!function_exists('tsml_db_set_address_approximate')) {
+  function tsml_db_set_address_approximate() {
+    $locations = tsml_get_locations();
+    $addresses_cache = get_option('tsml_addresses');
+
+    foreach ($locations as $location) {
+      // if location doesn't have the approximate tag
+      if (empty($location['address_approximate'])) {
+        $tmp_address = $location['formatted_address'];
+
+        // if the cached address has the approximate tag
+        if (!empty($addresses_cache[$tmp_address]['approximate'])) {
+          // Location in Database doesn't have approximate, and it's in the address cache, so write it to the database
+          update_post_meta($location['location_id'], 'approximate', $addresses_cache[$tmp_address]['approximate']);
```

Comment:
hey @brianw-area24 i think this may be setting `$location['approximate']` but checking for `$location['address_approximate']` above